SU-SOE/stanford_soe_helper
# 7.x 2.x spotlight image

# READY FOR REVIEW

# Summary
- Yo we haz Hi-res for spotlight image on list views!

# Needed By (Date)
- 

# Criticality
- 

# Steps to Test
- Update Image styles module with latest [PR](https://github.com/SU-SWS/stanford_image_styles/pull/42)
- Clear cache
- Navigate to /spotlight to confirm that the image in the list view are indeed 
`image-style-square-scale-and-crop-270-x-270`

# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2239
## Related PRs
https://github.com/SU-SWS/stanford_image_styles/pull/42
## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
